packages/router/src/entry.route.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/router
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Context} from '@loopback/core';
+
+/**
+ * An entry in the routing table
+ */
+export interface RouteEntry<P, A, R> {
```

Comment:
Let's use more descriptive names for these template args please, e.g.

```suggestion
export interface RouteEntry<PathType, Args, Retval> {
```